Jellyfin.Api/Controllers/FilterController.cs

```diff
@@ -92,7 +92,12 @@ public FilterController(ILibraryManager libraryManager, IUserManager userManager
                 }
             };
 
-            var itemList = ((Folder)item!).GetItemList(query);
+            var itemList = ((Folder?)item)?.GetItemList(query);
+            if (itemList == null)
+            {
+                return new QueryFiltersLegacy();
+            }
```

Comment:
```suggestion
            var folder = item as Folder;
            if (folder == null)
            {
                return new QueryFiltersLegacy();
            }

            var itemList = folder.GetItemList(query);
```
Something like this?